.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
```

Comment:
I noticed that `master.yml` has been removed. Do we no longer need to run the `release-drafter` action?

Reply:
Good catch, fixed!